native/src/main/java/io/ballerina/stdlib/sql/utils/Utils.java

```diff
@@ -428,6 +448,22 @@ private static PrimitiveTypeColumnDefinition generateColumnDefinition(SQLColumnM
                 ballerinaType);
     }
 
+    private static String getAnnotatedColumnName(StructureType streamConstraint, String fieldName) {
+        Object fieldAnnotationsObj = streamConstraint
+                .getAnnotation(fromString(RECORD_FIELD_ANN_PREFIX + fieldName));
+        if (fieldAnnotationsObj instanceof BMap) {
+            @SuppressWarnings("unchecked")
```

Comment:
Instead of having multiple `@SuppressWarnings` in the function body, shall we put a single annotation to the function signature?

eg: https://github.com/ballerina-platform/module-ballerina-grpc/blob/bc9e549ba180d4bd0c05425b8008529a6075ebec/native/src/main/java/io/ballerina/stdlib/grpc/nativeimpl/client/FunctionUtils.java#L100

Reply:
Fixed in [d0f16c7](https://github.com/ballerina-platform/module-ballerina-sql/pull/509/commits/d0f16c7c431693f9f5da817f39a52701a1056902)